### DISCUSSION TITLE
AI-900 credits

### DISCUSSION THREAD

**Discussion by mehboobpatel** [github/india#811] [General]
Hello official i have completed the ai4u course and having the certifications please would like to have the AI-900 credits as mentioned in the site please check the attachment i have attached my certificate 

![image](https://user-images.githubusercontent.com/92875632/158975804-6e6494ec-ccbc-4a6c-a0fd-892bf278291d.png)

[MicrosoftCertificate.pdf](https://github.com/github/india/files/8303220/MicrosoftCertificate.pdf)

**Comment by mehboobpatel** [comment]
Please answer 
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Reply by ey-mentor1** [reply]
Hello,
Since you have registered already, you can redeem by submitting the certificate.
In case you face issues, please drop mail to support team.
Thank you

**Reply by mehboobpatel** [reply]
where i should i submit the certificate i have attached the certificate and mail to support@frt 
still no response please check with it my mail id is masterpatel786@yahoo.in
